rmgpy/data/kinetics/family.py

```diff
@@ -1300,6 +1300,8 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
         for struct in productStructures:
             if isinstance(struct, Molecule):
```

Comment:
Why does this not need done for molecule objects?

Reply:
If you're referring to resetting ring membership, the `Molecule.update` method will also update the ring attributes.

Reply:
Ok